static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -299,11 +304,13 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
           projectId={currentProject.id}
           projectSlug={currentProject.slug}
           configType={
-            setupMode() === 'npm' || // switched to NPM option
-            (!setupMode() && defaultTab === 'npm') || // default value for FE frameworks when ?mode={...} in URL is not set yet
-            npmOnlyFramework // even if '?mode=jsLoader', only show npm instructions for FE frameworks
-              ? 'feedbackOnboardingNpm'
-              : 'replayOnboardingJsLoader'
+            backendPlatform
+              ? 'feedbackOnboardingCrashApi'
+              : setupMode() === 'npm' || // switched to NPM option
+                  (!setupMode() && defaultTab === 'npm') || // default value for FE frameworks when ?mode={...} in URL is not set yet
+                  npmOnlyFramework // even if '?mode=jsLoader', only show npm instructions for FE frameworks
+                ? 'feedbackOnboardingNpm'
+                : 'replayOnboardingJsLoader'
```

Comment:
`replayOnboardingJsLoader`, should be feedback specific?

Reply:
yes, it will be in the future, but we don't have those code snippets yet so i'm keeping it replay for now